src/Test.Utilities/CSharpProjectState.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Testing;
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    /// <summary>
+    /// This class is a copy-paste from
+    /// https://github.com/dotnet/roslyn-sdk/blob/master/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CSharp.Analyzer.Testing/CSharpProjectState.cs
```

Comment:
As explained in this comment this class should not really exists as it is purely a copy-paste from `Roslyn`. I guess that we need to update the testing dependency but I am not sure about the update politic for the branch. Could you please let me know if I can update the version? And also up to which version? Feel free to push a commit on this PR if you want.